src/models/enums/MessageCodes.ts

```diff
@@ -3,6 +3,9 @@ export enum MessageCodes {
 	MISSING_CONFIGURATION = "MISSING_CONFIGURATION",
 	INVALID_AUTH_CODE = "INVALID_AUTH_CODE",
 	SESSION_NOT_FOUND = "SESSION_NOT_FOUND",
+	PERSON_NOT_FOUND = "PERSON_NOT_FOUND",
+	SESSION_EXPIRED = "SESSION_EXPIRED",
```

Comment:
We already have `EXPIRED_SESSION`, reckon we can use that?

Reply:
Same with `STATE_MISMATCH` we already have `INCORRECT_SESSION_STATE`, could we use that?

Reply:
Removed EXPIRED_SESSION & STATE_MISMATCH